### PR TITLE
Sending token when it's presented in chart CR's annotations. 

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward giantswarm/giantswarm#13201

Follow up PR from https://github.com/giantswarm/app-operator/pull/570

## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/annotation/annotation.go [hunk 1]
```diff
@@ -23,5 +23,11 @@ const (
...
 	ValuesMD5Checksum = "chart-operator.giantswarm.io/values-md5-checksum"
 
-	Webhook = "chart-operator.giantswarm.io/webhook-url"
+	// WebhookURL is the name of the annotation storing an URL
+	// of the status webhook from app-operator.
+	WebhookURL = "chart-operator.giantswarm.io/webhook-url"
+
+	// WebhookURL is the name of the annotation storing an token which
+	// must be included when calling webhook URL.
+	WebhookToken = "chart-operator.giantswarm.io/webhook-token"
```
Comment (rossf7) [inline pkg/annotation/annotation.go:32]: Nitpick but same ordering problem.

File: service/controller/chart/resource/status/create.go [hunk 2]
```diff
@@ -137,6 +139,10 @@ func updateAppStatus(webhookURL string, status v1alpha1.ChartStatus, timeout tim
 		Version:      status.Version,
 	}
 
+	if token != "" {
```
Comment (rossf7) [inline service/controller/chart/resource/status/create.go:142]: Can you add a comment for this? To make it clearer why we need the token.

**Review by rossf7 — COMMENTED** [review]
Nice just a couple of comments.

**Comment by tomahawk28** [comment]
Closing this in favor of another PR

**Pull request closed without merging**